src/main/java/net/minecraftforge/common/FarmlandWaterManager.java

```diff
@@ -50,26 +55,19 @@
      * when the region this is unloaded (e.g. your TE is unloaded or the block is removed), and validate once it is loaded
      * @param world The world where the region should be marked. Only server-side worlds are allowed
      * @param ticket Your ticket you want to have registered
-     * @param chunkPoses The chunkPoses where the ticket is located
+     * @param masterChunk The chunk pos that is controls when the ticket may be unloaded. The ticket should originate from here.
+     * @param additionalChunks The chunks in that this ticket wants to operate as well.
      * @return The ticket for your requested region.
      */
     @SuppressWarnings("unchecked")
-    public static<T extends SimpleTicket<Vec3d>> T addCustomTicket(World world, T ticket, ChunkPos... chunkPoses)
+    public static<T extends SimpleTicket<Vec3d>> T addCustomTicket(World world, T ticket, ChunkPos masterChunk, ChunkPos... additionalChunks)
     {
```

Comment:
This is a Binary breaking change, and thus can not be accepted with this in place.

Reply:
It is not, a new method with the same signature as the old one has been added above